adapter_pipelines/cellranger/adapter.wdl

```diff
@@ -216,12 +241,23 @@ workflow Adapter10xCount {
       pipeline_tools_version = pipeline_tools_version
   }
 
+  # Rename analysis files so that all the file names are unique. For example, rename
+  # "${sample_id}/outs/raw_gene_bc_matrices/${reference}/barcodes.tsv" to "raw_barcodes.tsv" so that
+  # it does not overwrite "${sample_id}/outs/raw_gene_bc_matrices/${reference}/barcodes.tsv"
```

Comment:
I'm not sure I understand the comments here, so we have multiple `"${sample_id}/outs/raw_gene_bc_matrices/${reference}/barcodes.tsv"` files? If so, how could renaming them to `"raw_barcodes.tsv"` solves the problem?

Reply:
Oh I made a mistake copying the path! 

There is one file named "${sample_id}/outs/**filtered_gene_bc_matrices**/${reference}/barcodes.tsv"and another named "${sample_id}/outs/**raw_gene_bc_matrices**/${reference}/barcodes.tsv" and since they are both called "barcodes.tsv", this task renames the second one to "raw_barcodes.tsv"

Reply:
Ah, that makes more sense!